source/PlayerInfo.cpp

```diff
@@ -863,9 +879,12 @@ void PlayerInfo::BuyShip(const Ship *model, const string &name)
 		flagship.reset();
 		
 		// Record the transfer of this ship in the depreciation and stock info.
-		depreciation.Buy(*model, day, &stockDepreciation);
-		for(const auto &it : model->Outfits())
-			stock[it.first] -= it.second;
+		if(!gifting)
+		{
+			depreciation.Buy(*model, day, &stockDepreciation);
+			for(const auto &it : model->Outfits())
+				stock[it.first] -= it.second;
+		}
```

Comment:
Are ships that are gifts valued at 100% of the deprecation value, or at 25% of the deprecation value? (Or at some other value.)

Reply:
If a ship or outfit how no recorded depreciation, it is fully depreciated.
https://github.com/endless-sky/endless-sky/blob/8f29a0b3d1401d4fd0f4de6e7433867c9dbe7710/source/Depreciation.cpp#L86-L87